### PR TITLE
Config for v2.5.1 

### PR DESCRIPTION
**Pull request by JGuetschow**
No csv, html, image files in this commit so the merge request can be reviewed. I will push the changelog and the csv files to this branch and pr after review.

I accidentally committed the changes into main first, but reverted them in main and moved them to the proper branch.

(cherry picked from commit cdafdb95018b9df4ab1e6c296cd24bd38fcfecaf)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ To update to a new version, update
 - update the CSV to be downloaded into `primap-hist` in the Makefile,
 - update the filename and Zenodo record in `primap-hist/scripts/constants.py`
 - update changelog in `primap-hist/template.jinja2.html`
+- copy the replace the changelog files in `docs/changelog/` and rename the html file in the `html` directory to `index.html`.
```
Comment (rgieseke) [inline README.md]: Copy or replace? Something seems duplicated.

File: docs/primap-hist/index.html [hunk 2]
```diff
@@ -412,6 +411,14 @@ <h3>Contact</h3>
 
   <h3>Changelog</h3>
 
+  <h4>Version 2.5.1 (February 2024)</h4>
+  <p>
+    The v2.5.1 release updates mainly third party data. New and updated input data sources are UNFCCC data for AnnexI, CDIAC fossil fuel related emissions (@hefnerGlobalRegionalNational2023a), EDGAR v8.0 data (CO$_2$, CH$_4$, N$_2$O), cement process emissions data (@andrewGlobalCO2Emissions2023), and data from India's third National Communication. A more detailed chnagelog is available with the zenodo release of the dataset (<a class="primap-blue" href="https://doi.org/10.5281/zenodo.10705513">https://doi.org/10.5281/zenodo.10705513</a>). Additionally we provide detailed per country changes <a class="primap-blue" href="changelog/html/index.html">here</a>.
```
Comment (rgieseke) [inline docs/primap-hist/index.html]: Is there another step with replacements for the citation identifiers? If not, needs to be replaced I guess.

File: primap-hist/template.jinja2.html [hunk 3]
```diff
@@ -355,6 +355,14 @@ <h3>Contact</h3>
 
   <h3>Changelog</h3>
 
+  <h4>Version 2.5.1 (February 2024)</h4>
+  <p>
+    The v2.5.1 release updates mainly third party data. New and updated input data sources are UNFCCC data for AnnexI, CDIAC fossil fuel related emissions (@hefnerGlobalRegionalNational2023a), EDGAR v8.0 data (CO$_2$, CH$_4$, N$_2$O), cement process emissions data (@andrewGlobalCO2Emissions2023), and data from India's third National Communication. A more detailed chnagelog is available with the zenodo release of the dataset (<a class="primap-blue" href="https://doi.org/10.5281/zenodo.10705513">https://doi.org/10.5281/zenodo.10705513</a>). Additionally we provide detailed per country changes <a class="primap-blue" href="changelog/html/index.html">here</a>.
```
Comment (rgieseke) [inline primap-hist/template.jinja2.html]:
```suggestion
    The v2.5.1 release updates mainly third party data. New and updated input data sources are UNFCCC data for AnnexI, CDIAC fossil fuel related emissions (@hefnerGlobalRegionalNational2023a), EDGAR v8.0 data (CO$_2$, CH$_4$, N$_2$O), cement process emissions data (@andrewGlobalCO2Emissions2023), and data from India's third National Communication. A more detailed changelog is available with the Zenodo release of the dataset (<a class="primap-blue" href="https://doi.org/10.5281/zenodo.10705513">https://doi.org/10.5281/zenodo.10705513</a>). Additionally we provide detailed per country changes <a class="primap-blue" href="changelog/html/index.html">here</a>.
```

Comment (rgieseke) [reply primap-hist/template.jinja2.html]: I think this will then change the index file above.

**Review by lewisjared — APPROVED** [review]
Thanks @rgieseke! After @rgieseke suggestions and a rebuild it looks good.

File: docs/primap-hist/index.html (hunk 2 above)
Comment (JGuetschow) [reply docs/primap-hist/index.html]: For this bit it's done in the update changelog step. The appearances of the current citation are updated automatically from the config.

File: README.md (hunk 1 above)
Comment (JGuetschow) [reply README.md]: thanks, fixed it

File: docs/primap-hist/index.html (hunk 2 above)
Comment (rgieseke) [reply docs/primap-hist/index.html]: Neat!

Comment (rgieseke) [reply docs/primap-hist/index.html]: @JGuetschow Markdown `CO$2$` and citation (`@hefner ...`) identifiers are still there

Comment (JGuetschow) [reply docs/primap-hist/index.html]: darn. When I copied over the text I thought that I need to fix that. I'll fix it now (after a meeting(. thanks for spotting.

**Pull request merged**